exporter/exporterhelper/tracehelper_test.go

```diff
@@ -31,8 +31,8 @@ import (
 )
 
 const (
-	fakeReceiverName   = "fake_receiver_trace"
-	fakeExporterName   = "fake_exporter_trace"
+	fakeReceiverName   = "fake_receiver"
```

Comment:
These constants are for tracehelper_test. I would suggest make separate constants for metrics test.

Reply:
Fixed the receiver/exporter names. I will fix the span names in the other PR.